### PR TITLE
virtualenv + FP16 note added 

### PR DESCRIPTION
**Pull request by sraimund**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,36 +73,18 @@ Write `yes` and press ENTER to establish a first connection.
 
 ### For people using Python on a cluster node
 
-- create a **local** python environment using conda or pipenv. You need to ensure your environment is on the `/scratch/`, not `/home/`, to do so check the guides just below.
+- create a **local** python environment using conda, virtualenv or pipenv. 
+- **Note**: Before creating environments, you need to create a $USER folder on `/scratch` (`mkdir /scratch/$USER` and replace `$USER` with your username). If you do not have the rights to create the folder on `/scratch`, consider creating it in `/scratch/students/` (`mkdir /scratch/students/$USER`).
```
Comment (EmanuelaBoros) [inline README.md]: Students don't have the right to make a folder on scratch, they can only do it in scratch/students. Be clear about this distinction.

Comment (EmanuelaBoros) [reply README.md]: Keep in mind that this file is mostly for the students.

File: README.md [hunk 2]
```diff
@@ -116,9 +98,53 @@ conda create -n py310 python=3.10 anaconda
 ```
 In this example, the name of the environemnt is the same as the python version, but it have any name (depending on your needs).
```
Comment (EmanuelaBoros) [inline README.md]: I know that this was before, but could you correct it also please? Thank you

**Review by EmanuelaBoros — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sraimund) [reply README.md]: https://github.com/dh-epfl-students/student-project-howtos/pull/4/commits/cf2a85e17410de3ef350bd34fd408dfe1adac539

File: README.md (hunk 2 above)
Comment (sraimund) [reply README.md]: https://github.com/dh-epfl-students/student-project-howtos/pull/4/commits/b69c2fb02a144e48169ceffcab96e2bd56ba249d

**Pull request merged**